addons/website/views/snippets/s_mega_menu_odoo_menu.xml

```diff
@@ -2,10 +2,10 @@
 <odoo>
 
 <template id="s_mega_menu_odoo_menu" name="Odoo Menu" groups="base.group_user">
-    <section class="s_mega_menu_odoo_menu pt16 o_colored_level o_cc o_cc1">
+    <section class="s_mega_menu_odoo_menu pt16 o_colored_level o_cc o_cc1" data-vcss="001">
         <div class="container">
             <div class="row">
-                <div class="col-md-6 col-lg pt16 pb24">
+                <div class="col-lg pt16 pb24">
```

Comment:
Why changing this? And I am not sure it can be done in a stable commit :grimacing: